textures/metro_control_panel_001_pegboard_wip.py

```diff
@@ -0,0 +1,136 @@
+"""A collection of stuff related to https://stalburg.net/Body_message#Pegs."""
+from infra.encodings.binary import binary_decode
+from textures.metro_control_panel_001_pegboard import (metro_control_panel_001_pegboard, _invert_bool_list,
+                                                       _bools_to_binary_str, _bools_to_nor_mask, _pegs_to_bool_list)
+
+mp = metro_control_panel_001_pegboard
+
+
+if __name__ == "__main__":
+    def transform_pegs(a: str, mask: str, b: str) -> str:
```

Comment:
Adding some docs to each of the transformations what they actually do would be nice. All these functions look similar, too, maybe refactor it to extract common parts.